src/Adaptivity/AdaptedTriangulations.jl

```diff
@@ -58,6 +58,10 @@ function Geometry.get_glue(t::AdaptedTriangulation,::Val{d}) where d
   get_glue(t.trian,Val(d))
 end
 
+function Geometry.get_facet_normal(trian::AdaptedTriangulation)
+  get_facet_normal(trian.trian)
```

Comment:
The background triangulation of the returned `CellField` object would be `trian.trian`, correct? 

Shouldn't it be `trian` instead?

Reply:
No, `get_facet_normal` returns a lazy_array of fields. The lazy_array then gets wrapped into a CellField, which will have the desired triangulation. Note that I've redirected `get_facet_normal` instead of `get_normal_vector`.

Reply:
I.e we have this 

```julia
function get_normal_vector(trian::Triangulation)
  cell_normal = get_facet_normal(trian)
  get_normal_vector(trian,cell_normal)
end

function get_normal_vector(trian::Triangulation,cell_normal::AbstractArray)
  GenericCellField(cell_normal,trian,ReferenceDomain())
end
```

Reply:
ok, thanks, understood, ready to merge when the tests pass ...